stubs/core.stub

```diff
@@ -283,6 +283,17 @@ function str_ireplace($search, $replace, $subject, ?int &$count = null) {}
  */
 function stream_select(?array &$read, ?array &$write, ?array &$except, ?int $seconds, ?int $microseconds = null) {}
 
+/**
+ * @param int $error_code
+ * @param-out int $error_code
+ * @param string $error_message
+ * @param-out string $error_message
```

Comment:
What are these set to if the call does not fail?

Reply:
`$error_code` will be zero and `$error_message` an empty string: https://3v4l.org/muqgC